CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 - RestoreFrommnemonic: ported to Rust. Will now return UserAbortError on user abort instead of GenericError.
+- Added [rlp] as an optional dependency of the bitbox02 Python package using an environment marker
```

Comment:
maybe there should be  a separate changelog for the python package? after all it does not follow the same release schedule as the firmware, and has a different version, etc.